src/rust/engine/fs/store/src/immutable_inputs.rs

```diff
@@ -51,69 +51,6 @@ impl ImmutableInputs {
     self.0.workdir.path()
   }
 
-  /// Returns an absolute Path to immutably consume the given Digest from.
-  pub(crate) async fn path_for_file(
```

Comment:
Good news @stuhood you no longer have to help me de-duplicate this code :joy:

Reply:
Laziness pays off again!